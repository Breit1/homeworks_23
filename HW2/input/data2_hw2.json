{
    "user": {
        "region": "Moscow",
        "registered": "2014-12-24",
        "last_login": "2020-10-01",
        "email": "user@yahoo.com",
        "age": 39
    },
    "user2": {
        "region": "Bratsk",
        "registered": "2016-10-30",
        "last_login": "2021-11-21",
        "email": "user2@yahoo.com",
        "age": 35
    }
}