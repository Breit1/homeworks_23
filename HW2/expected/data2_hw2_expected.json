{
    "email_host_percentages": {
        "yahoo.com": 100.0
    },
    "registration_year_percentages": {
        "2016": 50.0,
        "2014": 50.0
    }
}